other/install.php

```diff
@@ -27,7 +27,7 @@
 
 // Database info.
 $databases = array(
-	'mysqli' => array(
+	'mysql' => array(
 		'name' => 'MySQLi',
```

Comment:
It should, I need to spin up a vm with MySQL on and test this. I'll get it done some time today and let you know when it's ready to commit to the upstream.